src/commands/dependabot/automerge.ts

```diff
@@ -0,0 +1,138 @@
+  '@salesforce/plugin-release-management',
+  'dependabot.consolidate'
+);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'dependabot.automerge');
+
+interface PullRequest {
+  state: string;
+  title: string;
+  user: {
+    login: string;
+  };
+  html_url: string;
+  number: number;
+  head: {
+    sha: string;
+  };
+}
+export default class AutoMerge extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+
+  public static readonly flagsConfig: FlagsConfig = {
+    owner: flags.string({
+      char: 'o',
+      description: messagesFromConsolidate.getMessage('owner'),
+    }),
+    repo: flags.string({
+      char: 'r',
+      description: messagesFromConsolidate.getMessage('repo'),
+      dependsOn: ['owner'],
+    }),
+    'max-version-bump': maxVersionBumpFlag,
+    dryrun: flags.boolean({
+      description: messagesFromConsolidate.getMessage('dryrun'),
+      char: 'd',
+      default: false,
+    }),
+  };
+
+  private octokit: Octokit;
+  private baseRepoObject: {
+    owner: string;
+    repo: string;
+  };
+
+  public async run(): Promise<void> {
+    const auth = ensureString(new Env().getString('GH_TOKEN'), 'GH_TOKEN is required to be set in the environment');
+    const { owner, repo } = await getOwnerAndRepo(this.flags.owner, this.flags.repo);
+
+    this.octokit = new Octokit({ auth });
+    this.baseRepoObject = {
+      owner,
+      repo,
+    };
+
+    const eligiblePRs = (
+      await this.octokit.request('GET /repos/{owner}/{repo}/pulls', this.baseRepoObject)
+    ).data.filter(
+      (pr) =>
+        pr.state === 'open' &&
+        pr.user.login === 'dependabot[bot]' &&
+        meetsVersionCriteria(pr.title, this.flags['max-version-bump'])
+    ) as PullRequest[];
+
+    const greenPRs = (await Promise.all(eligiblePRs.map((pr) => this.isGreen(pr)))).filter((pr) => pr !== undefined);
+    const mergeablePRs = (await Promise.all(greenPRs.map((pr) => this.isMergeable(pr)))).filter(
+      (pr) => pr !== undefined
+    );
+
+    this.ux.table(mergeablePRs, {
+      columns: [
+        { key: 'title', label: 'Green, Mergeable PR' },
+        { key: 'html_url', label: 'Link' },
+      ],
+    });
+    this.ux.log('');
+
+    if (mergeablePRs.length === 0) {
+      this.ux.log('No PRs can be automerged');
+      return;
+    }
+
+    const prToMerge = mergeablePRs[0];
+    this.ux.log(
+      `will merge PR ${prToMerge.number.toString()} | ${prToMerge.title} if not dryrun [${
+        this.flags.dryrun as string
+      }] is not true`
+    );
+
+    if (this.flags.dryrun === false) {
+      this.ux.log('starting merge');
+      const mergeResult = await this.octokit.request('PUT /repos/{owner}/{repo}/pulls/{pull_number}/merge', {
+        ...this.baseRepoObject,
+        pull_number: mergeablePRs[0].number,
+      });
+      this.ux.logJson(mergeResult);
+    }
```

Comment:
```suggestion

    if (this.flags.dryrun === false) {
      this.ux.log(`merging ${prToMerge.number.toString()} | ${prToMerge.title}`);
      const mergeResult = await this.octokit.request('PUT /repos/{owner}/{repo}/pulls/{pull_number}/merge', {
        ...this.baseRepoObject,
        pull_number: prToMerge.number,
      });
      this.ux.logJson(mergeResult);
    } else {
       this.ux.log(`dry run ${prToMerge.number.toString()} | ${prToMerge.title}`);
    }
```

Reply:
if you're doing a dry run to see what all PRs are available to automerge, this lets you see which one would be executed before actually doing it.  Without the 100-103, it's not clear what's about to happen if you remove --dryrun